frontend/components/markup-modal/markup-modal-content.js

```diff
@@ -16,7 +16,8 @@ export default function MarkupModalContent({ example, baseUrl }) {
     documentation,
     example: { pattern, namedParams, queryParams },
     link,
-    preview: { style: initialStyle, buildFromExample } = {},
+    preview: { style: initialStyle } = {},
+    realBadge,
```

Comment:
Would it be clearer to call this property `isBadgeSuggestion`?

Reply:
It would be :-) Fixed.